### PR TITLE
Add font size and font familly option for selection

### PR DESCRIPTION
**Pull request by davidbonan**
@vjeux @paulomenezes 
New PR from the last master.

Add editable font size and font familly on select text

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -452,6 +465,60 @@ export class App extends React.Component<{}, AppState> {
...
+                        element.font = `${value}px ${element.font.split("px ")[1]}`;
+                        this.redrawTextBoundingBox(element);
+                      }
+                    })
+                  }
+                />
+                <h5>Font familly</h5>
+                <ButtonSelect 
+                  options={[
+                    {value: "Virgil", text: "Virgil"},
```
Comment (vjeux) [inline src/index.tsx]: Could you remove everything but Virgil, Helvetica and Courier, the rest don't seem to provide a ton of value to be different

File: src/index.tsx [hunk 2]
```diff
@@ -452,6 +465,60 @@ export class App extends React.Component<{}, AppState> {
               </>
             )}
 
+            {hasText(elements) && (
+              <>
+                <h5>Font size</h5>
+                <ButtonSelect
+                  options={[
+                    { value: 16, text: "16 px" },
```
Comment (vjeux) [inline src/index.tsx]: Could you pick 4 and name them "Small", "Medium", "Large", "Very Large" or something. I'd rather have fewer options if possible

Comment (davidbonan) [reply src/index.tsx]: Great idea !!! it's done

File: src/index.tsx (hunk 1 above)
Comment (davidbonan) [reply src/index.tsx]: It's done

**Comment by vjeux** [comment]
Thank you so much!

**Pull request merged**